file_store/src/file_sink.rs

```diff
@@ -63,28 +67,37 @@ pub struct FileSinkBuilder {
     max_size: usize,
     roll_time: Duration,
     deposits: Option<file_upload::MessageSender>,
+    file_upload: Option<FileUpload>,
     auto_commit: bool,
     metric: &'static str,
-    shutdown_listener: triggered::Listener,
 }
 
 impl FileSinkBuilder {
-    pub fn new(
-        prefix: impl ToString,
-        target_path: &Path,
-        metric: &'static str,
-        shutdown_listener: triggered::Listener,
-    ) -> Self {
+    pub fn new(prefix: impl ToString, target_path: &Path, metric: &'static str) -> Self {
+        Self {
+            prefix: prefix.to_string(),
+            target_path: target_path.to_path_buf(),
+            tmp_path: target_path.join("tmp"),
+            max_size: 50_000_000,
+            roll_time: Duration::minutes(DEFAULT_SINK_ROLL_MINS),
+            deposits: None,
+            file_upload: None,
+            auto_commit: true,
+            metric,
+        }
+    }
+
+    pub fn new_tm(prefix: impl ToString, target_path: &Path, metric: &'static str) -> Self {
```

Comment:
`new` and `new_tm` seem to be identical; are they both needed?

Reply:
Fixed, there were different until the a few commits ago.  Forgot to update these, addressed here 3f8c94b66b069cdb497c986892a97a6fe909d47d